adb_graphics/figures/maps.py

```diff
@@ -86,15 +88,16 @@ class Map():
                         TILE_DEFS dictionary
     '''
 
-    def __init__(self, airport_fn, ax, **kwargs):
+    def __init__(self, airport_fn, ax, field, **kwargs):
```

Comment:
Need to include field here so we can turn on/off airports.

Reply:
Good point in not passing the field.  This has been changed to pass a boolean flag for plot_airports.